system/security/securesocket/securesocket.cpp

```diff
@@ -1631,6 +1874,18 @@ SECURESOCKET_API ISecureSocketContext* createSecureSocketContextSecret(const cha
         return createSecureSocketContext(sockettype);
 }
 
+SECURESOCKET_API ISecureSocketContext* createSecureSocketContextSecretSrv(const char *mtlsSecretName)
+{
+    if (!queryMtls())
+        throw makeStringExceptionV(-100, "TLS secure communication requested but not configured");
+
+    IPropertyTree *info = queryMtlsSecretInfo(mtlsSecretName);
+    if (info)
+        return createSecureSocketContextEx2(info, ServerSocket);
+    else
+        throw makeStringExceptionV(-101, "TLS secure communication requested but not configured");
```

Comment:
minor: no vargs so should use 'makeStringException' (and same on line 1880)

Reply:
Yes, fixed